scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -31,88 +31,86 @@
 {% endblock %}
 
 {% block content_main %}
-<form>
```

Comment:
Looks like it's a lot of changes, but the only difference with master is that I've removed this form tag, and the form closing one. Then I've outdented the rest of the code, and that's the differences you see now!

Reply:
:+1: